Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -439,6 +439,14 @@ Description of keywords in syntax:
    FlexForm fields always end with a "regular value" of course.
 
 
+.. caution::
+   Modifying the :sql:`sys_file` table using DataHandler is blocked since TYPO3
+   version 8.7.57, 9.5.46, 10.4.43, 11.5.35, 12.4.11, and 13.0.1. The table
```

Comment:
I suggest to use a `.. versionchanged:: 13.0.1/12.4.11/...` directly after the "caution" directive, this is the default way to communicate changes in specific versions.

I am not sure that we should add ELTS versions in the recent docs, what do you think @linawolf ?